onnxruntime/core/mlas/lib/erf.cpp

```diff
@@ -64,7 +64,7 @@ MLAS_INTERNAL_DATA const struct {
     float Exp_C;
     int32_t Exp_X7F;
 } MlasErfConstants = {
-    3.725f,
+    3.925f,
```

Comment:
What's the reason for this change? Is there any documentation or comments that explain how you'd decide a different value is needed and what an appropriate one is?

Reply:
This value is ErfUpperAbsRange. For values whose abs is greater than ErfUpperAbsRange, it will be treated as ErfUpperAbsRange. The result of Erf(ErfUpperAbsRange) should be 1, but 3.725 can only give a value very close to 1. 3.925 is the appropriate value can get 1.